build

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+#
+# Build steps to validate client_wrapper.
+
+( # Run unit tests and calculate code coverage.
+  coverage run --source client_wrapper -m unittest discover
+) &&
+( # Check that source has correct formatting.
+  yapf --diff --recursive --style google ./
+) &&
+( # Run static analysis for Python bugs/cruft.
+  pyflakes client_wrapper/*.py tests/*.py
+)
```

Comment:
Is there a way to do this but keep the comments around? That's what tripped me up.

I tried this:
```
# Run unit tests and calculate code coverage.
coverage run --source client_wrapper -m unittest discover && \
 \ # Check that source has correct formatting.
yapf --diff --recursive --style google ./ && \
 \ # Run static analysis for Python bugs/cruft.
pyflakes client_wrapper/*.py tests/*.py
```

But then it fails:

```
$ ./build 
........
----------------------------------------------------------------------
Ran 8 tests in 0.489s

OK
./build: line 7:  #: command not found
./build: line 9:  #: command not found
```